jina/drivers/search.py

```diff
@@ -155,12 +155,12 @@ def _fill_matches(self, doc, op_name, topks, scores, topk_embed):
         else:
             for idx, (numpy_match_id, score) in enumerate(zip(topks, scores)):
                 vector = None
-                if topk_embed is not None:
+                if topk_embed[idx] is not None:
```

Comment:
I wonder if this could work if topk_embed is `none`?
Will it get this?
`TypeError: 'NoneType' object is not subscriptable`

Reply:
it seems not to be the case, as it is not considerrd in the case with dense embeddings

Reply:
So this won't happen? topk_embed won't be None but the elements could be None

Reply:
yes, see the caller